posthog/migrations/0344_add_new_export_type.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.19 on 2023-08-11 11:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0343_team_has_completed_onboarding_for"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="batchexportdestination",
+            name="type",
+            field=models.CharField(
+                choices=[("S3", "S3"), ("Snowflake", "Snowflake"), ("Postgres", "Postgres")],
```

Comment:
should we just add the future expected ones now to not create too many django migrations?

Reply:
Don't see why, it's confusing to add support for choices in the model that are not supported. Moreover, this migration doesn't actually touch the db as we are only adding a choice.